tensorflow/python/ops/signal/fft_ops.py

```diff
@@ -324,6 +325,66 @@ def _grad(op, grad):
 
   return _grad
 
+@tf_export("signal.fftshift")
+def fftshift(x, axes=None):
+    """Shift the zero-frequency component to the center of the spectrum.
+    This function swaps half-spaces for all axes listed (defaults to all).
+    Note that ``y[0]`` is the Nyquist component only if ``len(x)`` is even.
+
+    Args:
+      x: `Tensor`, input tensor.
+      axes: `int` or shape `tuple`, optional
+             Axes over which to shift.  Default is None, which shifts all axes.
+
+    Returns:
+      `Tensor`, The shifted tensor.
+
+    Example:
+      >>> fftshift([ 0.,  1.,  2.,  3.,  4., -5., -4., -3., -2., -1.])
+      <tf.Tensor: id=3, shape=(10,), dtype=float32,numpy=array
+      ([-5., -4., -3., -2., -1.,  0.,  1.,  2.,  3.,  4.], dtype=float32)>
+    """
+    x = _ops.convert_to_tensor_v2(x)
+    if axes is None:
+        axes = tuple(range(x.ndim))
+        shift = [dim // 2 for dim in x.shape]
+    elif isinstance(axes, int):
+        shift = x.shape[axes] // 2
+    else:
+        shift = [x.shape[ax] // 2 for ax in axes]
+
+    return manip_ops.roll(x, shift, axes)
+
+
+@tf_export("signal.ifftshift")
+def ifftshift(x, axes=None):
+    """The inverse of fftshift. Although identical for even-length x,
+    the functions differ by one sample for odd-length x.
+
```

Comment:
could you please add a compatibility declaration for both? e.g. 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/signal/dct_ops.py#L64-L67

```python
  @compatibility(numpy)
  Equivalent to numpy.fft.fftshift.
  @end_compatibility
```